### PR TITLE
Pin cli plugins

### PR DESCRIPTION
**Pull request by cprovencher**
https://jira.mesosphere.com/browse/DCOS-57266

make CLI instantiation a fixture

### CODE REVIEW

**Comment by cprovencher** [comment]
@mesosphere-mergebot label ready for review

File: packages/dcos-integration-test/extra/conftest.py [hunk 1]
```diff
@@ -169,3 +171,28 @@ def _dump_diagnostics(request, dcos_api_session):
...
+
+
+@pytest.fixture(scope='session')
+def install_dcos_cli(tmpdir_factory: TempdirFactory):
+    """
+    Install the CLI.
+    """
+    tmpdir = tmpdir_factory.mktemp('dcos_cli')
+    cli = dcos_cli.DcosCli.new_cli(
+        download_url='https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.11/dcos',
```
[the remark this answers is not in the document]
Comment (bamarni) [reply packages/dcos-integration-test/extra/conftest.py]:
```suggestion
        download_url='https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/0.8.0/dcos',
```

File: packages/dcos-integration-test/extra/conftest.py [hunk 2]
```diff
@@ -169,3 +171,28 @@ def _dump_diagnostics(request, dcos_api_session):
...
+@pytest.fixture(scope='session')
+def install_dcos_cli(tmpdir_factory: TempdirFactory):
+    """
+    Install the CLI.
+    """
+    tmpdir = tmpdir_factory.mktemp('dcos_cli')
+    cli = dcos_cli.DcosCli.new_cli(
+        download_url='https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.13/dcos',
+        core_plugin_url='https://downloads.dcos.io/cli/releases/plugins/dcos-core-cli/linux/x86-64/dcos-core-cli-1.13-patch.5.zip',  # noqa: E501
+        ee_plugin_url='https://downloads.dcos.io/cli/releases/plugins/dcos-enterprise-cli/linux/x86-64/dcos-enterprise-cli-1.13-patch.1.zip',  # noqa: E501
```
Comment (bamarni) [inline packages/dcos-integration-test/extra/conftest.py]:
```suggestion
        ee_plugin_url='https://downloads.mesosphere.io/cli/releases/plugins/dcos-enterprise-cli/linux/x86-64/dcos-enterprise-cli-1.13-patch.1.zip',  # noqa: E501
```

**Comment by cprovencher** [comment]
@mesosphere-mergebot help

**Comment by cprovencher** [comment]
@mesosphere-mergebot changelog-not-required minor CLI pin for CI

**Comment by cprovencher** [comment]
@mesosphere-mergebot label ship it

**Pull request merged**